wolox-books/src/app/user/user.ts

```diff
@@ -0,0 +1,10 @@
+export class User {
+  constructor(private firstName: string, private lastName: string, private email: string, private password: string, private confirmPassword: string) {
```

Comment:
There's no actual need for you to declare the user as a full TS class. You can use it as a model and that way use it as a type for type checking, but no need to actually using it as a full class